rate: 10
spawn_height: 0.06

car:
  # init_pose:
  #   pos: [0.5, 0.0, 0.06]
  sensor_noise:
    pos: [0.01, 0.01, 0.00]
    orient: [0.0, 0.0, 0.01]

ball:
  init_speed: 0.0
  # init_pose:
  #   pos: [0.0, 0.5, 0.06]
  #   orient: [0.0, 0.0, 0.0]
  sensor_noise:
    pos: [0.0, 0.0, 0.0]
    orient: [0.0, 0.0, 0.0]
